netlify.toml

```diff
@@ -154,7 +159,9 @@ package = "./_plugins/netlify-plugin-health-check"
   CRDS_SEARCH_ENDPOINT = "crds-search.crossroads.net"
   CRDS_GROUP_LEADER_DOMAIN = "group-leader.crossroads.net"
   CRDS_GROUPS_ENDPOINT = "groups.crossroads.net"
+  CRDS_GROUPS_LEGACY_ENDPOINT = "groups-legacy.crossroads.net"
   CRDS_INVOICES = "invoices.crossroads.net"
+  CRDS_CORKBOARD_ENDPOINT = "crds-angular.crossroads.net/corkboard"
```

Comment:
Looks like this variable is only defined in the master environment. Does this need to exist in other environments?